dags/full_scrape.py

```diff
@@ -0,0 +1,71 @@
+import os
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.operators.python_operator import BranchPythonOperator
+from airflow.operators.dummy_operator import DummyOperator
+
+from dags.constants import LA_METRO_DATABASE_URL, AIRFLOW_DIR_PATH, \
+    DAG_DESCRIPTIONS, START_DATE
+from operators.blackbox_docker_operator import BlackboxDockerOperator
+
+
+default_args = {
+    'start_date': START_DATE,
+    'execution_timeout': timedelta(hours=12),
+}
+
+default_docker_args = {
+    'image': 'datamade/scrapers-us-municipal',
+    'volumes': [
+        '{}:/app/scraper_scripts'.format(os.path.join(AIRFLOW_DIR_PATH, 'dags', 'scripts'))
+    ],
+    'command': 'scraper_scripts/targeted-scrape.sh',
+}
+
+docker_base_environment = {
+    'DECRYPTED_SETTINGS': 'pupa_settings.py',
+    'DESTINATION_SETTINGS': 'pupa_settings.py',
+    'DATABASE_URL': LA_METRO_DATABASE_URL,  # For use by entrypoint
+    'LA_METRO_DATABASE_URL': LA_METRO_DATABASE_URL,  # For use in scraping scripts
+    'RPM': 60,
+}
+
+with DAG(
```

Comment:
Note that I could revise the config object structure and scrape factory to allow for DAGs with multiple tasks in order to cover the full scrape and hourly ETL, but I actually like the simplicity they have now, so I'd prefer to explicitly define the DAGs with multiple tasks, for now.